.github/workflows/test_exporters_common.yml

```diff
@@ -36,7 +36,8 @@ jobs:
         run: |
           pip install --upgrade pip
           pip install --no-cache-dir torch torchvision torchaudio --index-url https://download.pytorch.org/whl/cpu
-          pip install .[tests,exporters]
+          pip install optimum-onnx@git+https://github.com/huggingface/optimum-onnx.git
+          pip install .[tests]
```

Comment:
To fix circular dependency issue
Using pip install .[onnx] results in a dependency graph where optimum@PR points to optimum-onnx@main which points at optimum@main